CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Contributing to the Coda Pack SDK
+
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+Changes to the core SDK itself can only be made by Coda engineers, and they must be coordinated with changes to the core Coda product. We do however welcome contributions to the [SDK documentation][docs], which can be done without access to or knowledge of the private codebase.
+
+
+## Documentation changes
+
+For small contributions, such fixing a typo or adding a clarifying sentence, you can directly submit a pull request to this repo. For larger changes, such as adding a new page or code sample, first make a post to the [Coda Community][community] with your intentions to get feedback from a Codan. This helps ensure that you don't waste effort for a change which may not be approved.
+
+
+### Style guide
+
+When contributing to the documentation, please ensure your changes comply with the [style guide][style_guide]. Some elements of the style guide are enforced automatically using lint rules, but many others must be caught manually.
+
+
+### Environment setup
+
+While some documentation changes require only a single edit, others require building or validating your changes using the scripts in this repo's `Makefile`. This build system is designed to work on Unix-like command lines (Linux, Mac OSX, etc) and has a lot of dependencies.
+
+An easy way to get setup is to use [Google Cloud Shell][cloud_shell], an hosted command-line environment and web IDE that comes with most of the dependencies already installed. You can launch Google Cloud Shell and clone this repo using the button below:
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://shell.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/coda/packs-sdk.git&cloudshell_workspace=.&cloudshell_open_in_editor=docs/index.md)
+
+The only dependency that needs to be installed manually in Google Cloud Shell is `pipenv`:
+
+```sh
+pip install --user pipenv
+export PATH=$HOME/.local/bin:$PATH
+```
+
+You can then install the other dependencies using the Makefile using the bootstrap script:
+
+```sh
+make bs
```

Comment:
Nit, should we just say `make bootstrap` here to avoid conflating with the other colloquial meaning of "BS"?

Reply:
Yes, much safer this way :-)